.github/workflows/manual.yml

```diff
@@ -0,0 +1,66 @@
+# This is a basic workflow that is manually triggered
+
+name: Build and Publish
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Version'
+        # Default value if no value is explicitly provided
+        default: 'RORY-SNAPSHOT'
+        # Input has to be provided for the workflow to run
+        required: true
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
```

Comment:
Is there a reason you use ubuntu-20.04 over ubuntu-latest?

Reply:
I probably copied it from somewhere, I'll fix it. Thanks!